Sources/Amplitude/Utilities/EventPipeline.swift

```diff
@@ -12,13 +12,16 @@ public class EventPipeline {
     let storage: Storage?
     let logger: (any Logger)?
     let configuration: Configuration
+    var maxRetryInterval: TimeInterval = 60
+    var maxRetryCount: Int = 6
```

Comment:
@crleona Do you think these two configurations need to be placed in the Configuration?